src/components/DataDecorator/DataDecorator.js

```diff
@@ -288,6 +314,7 @@ DataDecorator.defaultProps = {
   stopPropagationEvents: undefined,
   scoreDescription: (score, scoreThresholds) =>
     `Score ${score} out of ${scoreThresholds.slice(-1)[0]}`,
+  midLineTruncation: {},
```

Comment:
If going with an `enabled` prop, perhaps we can provide healthy defaults so it requires less configuration from a consumer's end? For example, these defaults could be the props outlined below in `DataDecorator.stories.js`:

https://github.com/carbon-design-system/ibm-security/pull/774/files#diff-e5a99da599b761a0caca127c61f54c52a1eac4f10da8af6c55c24d9770d2101fR54-R56

Reply:
fixed